backend/app/api/api_v1/endpoints/health.py

```
@@ -0,0 +1,104 @@
+    Returns 200 only if the application can serve traffic.
+    Used by: Kubernetes readiness probes, load balancers.
+    """
+    try:
+        # Simple query to verify database connection
+        await db.execute(text("SELECT 1"))
+        return {
+            "status": "ready",
+            "database": "connected",
+            "timestamp": datetime.utcnow().isoformat()
+        }
+    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)